server/middleware/auth.coffee

```diff
@@ -310,17 +311,18 @@ module.exports =
       emails.recruitNotes ?= {}
       emails.recruitNotes.enabled = false
       msg = "Unsubscribed #{email} from recruiting emails."
+      yield user.update {$set: {emails: emails}}
     else if req.query.employerNotes
       emails.employerNotes ?= {}
       emails.employerNotes.enabled = false
       msg = "Unsubscribed #{email} from employer emails."
+      yield user.update {$set: {emails: emails}}
```

Comment:
Why are we calling this twice now for these two if clauses?

Still calling below on line 327.

Reply:
Good call, refactored, then unrefactored partially.